frontend/public/components/monitoring/targets.tsx

```diff
@@ -211,7 +211,7 @@ const getRowProps = (target: Target) => ({ id: target.scrapeUrl, title: target.l
 const List: React.FC<ListProps> = ({ loaded, loadError, targets }) => {
   const { t } = useTranslation();
 
-  const filters = useSelector(({ k8s }: RootState) => k8s.getIn([REDUX_ID, 'filters']));
+  const filters = useSelector(({ sdkK8s }: RootState) => sdkK8s.getIn([REDUX_ID, 'filters']));
```

Comment:
I believe it was just changed back to `k8s` in #10714

cc @florkbr @andrewballantyne

Reply:
Yup, it just merged, we are back to `k8s` for everything -- we switched it back because it was having unintended side-effects to the code base & 4.10. We need to write a better layer between the redux store and the code that needs the data; the SDK will attempt to do this moving forward.